app/models/subnet.rb

```diff
@@ -111,6 +111,15 @@ class Jail < ::Safemode::Jail
           :vlanid, :mtu, :nic_delay, :boot_mode, :dhcp?, :nil?, :has_vlanid?, :dhcp_boot_mode?, :description, :present?
   end
 
+  def self.network_reorder(order_string = 'network')
+    adapter = connection.adapter_name.downcase
+    if adapter.starts_with?('postgresql')
+      self.reorder(order_string.sub('network', 'inet(network)'))
```

Comment:
Given we're now PostgreSQL-only, should we actually make a migration that uses the inet type for the column?

Reply:
don't we need to still support sqlite for ci?

Reply:
Yes, but you can have a conditional in the migration. You'd end up with the same difference that you have now. The only thing I'd be worried about is that it might serialize to a different object (`IPAddr` for example) and the code would behave differently. That's why I won't insist on it, but would ask you to consider it.

@tbrisker what do you think?

Reply:
Let's leave that till later, but long term I think it would be a good improvement. We can even drop the separate cidr field since PostgreSQL can store it in a single field.